### PR TITLE
fix nproc

### LINKED ISSUE #203: fix number of processors used to build TP
we're telling people to use 10 cores, but not everyone has 10 cores available

we should refine those steps to tell people to use however many cores are available on their setup

### PR DESCRIPTION
**Pull request by plebhash**
close #203

### CODE REVIEW

File: src/getting-started.md [hunk 1]
```diff
@@ -82,13 +82,15 @@ git clone https://github.com/Sjors/bitcoin.git
...
-Next, compile the template provider:
+Next, compile the template provider. You can accelerate the process by using multiple cores.
 
 ```bash
 make clean
-./autogen.sh && ./configure 
-make -j 10
+./autogen.sh && ./configure
+nproc # find out how many cores are available on your CPU
+make -j <nproc> # replace the result of nproc here
```
Comment (pavlenex) [inline src/getting-started.md:91]: Shouldn't we just give an example or a good default?

Comment (plebhash) [reply src/getting-started.md:91]: I'll update it to automate this

Comment (plebhash) [reply src/getting-started.md]: https://github.com/stratum-mining/stratumprotocol.org/pull/204/commits/c406f7a85937f2bf74dfba9d02a38bcd6c0c1a42 lmk what you think @pavlenex

Comment (pavlenex) [reply src/getting-started.md]: Sounds good but I have yet to test it.

**Comment by pavlenex** [comment]
@GitGab19 can you review this one when you get a chance please?

File: src/getting-started.md [hunk 2]
```diff
@@ -82,13 +82,16 @@ git clone https://github.com/Sjors/bitcoin.git
 cd bitcoin
 git checkout sv2
 ```
-Next, compile the template provider:
+Next, compile the template provider. You can accelerate the process by using multiple cores.
```
Comment (plebhash) [inline src/getting-started.md:85]:
```suggestion
Next, compile the template provider. You can accelerate the process by using multiple cores with the `-j` option.
```

**Review by GitGab19 — APPROVED** [review]
Should we mention the binaries for the TP provided by Sjors in our guide?
https://github.com/Sjors/bitcoin/releases/tag/sv2-tp-0.1.2

**Comment by plebhash** [comment]
> https://github.com/Sjors/bitcoin/releases/tag/sv2-tp-0.1.2

could be a good alternative to making people build their own TP, perhaps a new PR going in this direction could replace this one

**Comment by GitGab19** [comment]
> > Should we mention the binaries for the TP provided by Sjors in our guide?
> > https://github.com/Sjors/bitcoin/releases/tag/sv2-tp-0.1.2
> 
> could be a good alternative to making people build their own TP, perhaps a new PR going in this direction could replace this one
> 
> but I would point them to the release page (as a general step), not one specific release
> 
> that would avoid the need for future PRs updating the instructions whenever new releases come out

Yeah I totally agree on the approach 👍

**Comment by pavlenex** [comment]
Agreed it is easier with TP binaries. Do we want to do those TP changes in this PR @plebhash ?

**Comment by plebhash** [comment]
> Agreed it is easier with TP binaries. Do we want to do those TP changes in this PR @plebhash ?

done

**Review by pavlenex — APPROVED** [review]

**Review by GitGab19 — APPROVED** [review]

**Comment by plebhash** [comment]
@pavlenex @GitGab19 are you guys familiar with this CI error?

**Comment by pavlenex** [comment]
@plebhash Yes linkcheck, Twitter blocks it so it fails, it's always safe to proceed if linkcheck fails at twitter links.

**Review by pavlenex — APPROVED** [review]

**Pull request merged**